editor/configs/unitColumns.ts

```diff
@@ -35,6 +36,11 @@ export const ALL_UNIT_COLUMNS = {
   veh_mod_year: {
     path: "veh_mod_year",
     label: "Year",
+    editable: true,
+    inputType: "number",
+    inputOptions: {
+      validate: commonValidations.isNumber,
```

Comment:
like john mentioned in [this comment](https://github.com/cityofaustin/vision-zero/pull/1702#discussion_r1956563098), now you cant save this field when its blank. do we need to fix that in a separate issue before rolling this out? it seemed out of scope to address that in this PR

Reply:
yea I guess if someone made a mistake and wanted to go back to null? seems edge case but idk

i wonder if some day it would make sense to hook in zod and be able chain validations but that could be over the top. definitely feels beyond the scope here to me!

Reply:
adding in zod is a cool idea! now that Mateo built RHF into our inputs it should be straightforward to use the Zod adapater, i think 🤩 

i agree that we should enable this field to be blank and that it's an edge case. i have an issue floating around to apply missing validations to a handful of fields. will add this to it.

* https://github.com/cityofaustin/atd-data-tech/issues/20468

what you have here is good for now 🚢